dashboard.csss-ci.com/server/api/models/Suppliers_model.php

```diff
@@ -31,7 +46,11 @@ public function login($where, $orWhere) {
                         ->get()
                         ->result();
     }
-
+    /**
+     * gets all of the rows in query
+     *@param null
```

Comment:
Don't need the this (just write)
/**
* gets all the rows in the query
* @ returns
*/